ios/brave-ios/Sources/BraveWallet/Crypto/Stores/SendTokenStore.swift

```diff
@@ -528,6 +568,11 @@ public class SendTokenStore: ObservableObject, WalletObserverStore {
     addressError = sendAddress.isFILAddress ? nil : .notFilAddress
   }
 
+  private func validateBitcoinSendAddress(fromAccount: BraveWallet.AccountInfo) {
+    let isMainnet = fromAccount.keyringId == BraveWallet.KeyringId.bitcoin84
+    addressError = sendAddress.isBTCAddress(isMainnet: isMainnet) ? nil : .notBtcAddress
+  }
```

Comment:
Can we tie this somehow not to account's keyring, to current token's network/chain? I'm going to extend bitcoin support with imported accounts which will introduce new types of btc keyrings and this(and similar) validation should just work.